spec/requests/political_edit_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+RSpec.describe "Documents" do
```

Comment:
I'd include this in the documents request spec. We hit a bit of a naming problem that I looked at earlier: https://github.com/alphagov/content-publisher/pull/1507